stl/inc/format

```diff
@@ -3,6 +3,37 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 

```

Comment:
We also need to record libfmt's latest commit in `docs/cgmanifest.json`.

Reply:
do we? I wasn't sure, since we're not using the directly, but rather deriving from it (this makes the commit hash a little meaningless)

Reply:
allright, added.

Reply:
According to my understanding, it is necessary.